node/request.js

```diff
@@ -195,20 +204,28 @@ TChannelRequest.prototype.shouldRetry = function shouldRetry(err, res, arg2, arg
         return false;
     }
 
+    if (self.options.retryFlags.never) {
+        return false;
+    }
+
     if (err) {
         switch (err.type) {
             case 'tchannel.bad-request':
             case 'tchannel.canceled':
                 return false;
 
-            case 'tchannel.socket':
-            case 'tchannel.timeout':
             case 'tchannel.busy':
             case 'tchannel.declined':
```

Comment:
The busy/declined path, it's hardwired to true here.

Seems a bit of a minor miss to me:
- would be better explicate a flag, say `"d"` for declined
- change the default retryflags then to be `"cd"`

We may not feel the lack of it much, but implementing the code seemed to point it out to me as an oversight.